src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,75 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
 
     public void getStatistic(String fromFileName, String toFileName) {
+        String[] data = readFromFile(fromFileName);
+        if (data != null) {
+            int countSupply = 0;
+            int countBuy = 0;
+            for (String dataLine : data) {
+                int indexOfComa = dataLine.indexOf(',') + 1;
+                if (dataLine.contains("supply")) {
+                    countSupply += Integer.parseInt(dataLine.substring(indexOfComa));
+                }
+                if (dataLine.contains("buy")) {
+                    countBuy += Integer.parseInt(dataLine.substring(indexOfComa));
+                }
+            }
+            String result = "supply,"
+                    + countSupply
+                    + '/'
+                    + "buy,"
+                    + countBuy
+                    + '/'
```

Comment:
let's separate with `System.lineSeparator()` and pass to `writeToFile` a ready to use string, not an array